.github/workflows/ci.yml

```diff
@@ -27,7 +27,11 @@ jobs:
     strategy:
       # Test for several versions of TFX in parallel
       matrix:
-        depconstraint: ["tfx~=1.0.0 tensorflow~=2.5.0", "tfx~=1.3.0 tensorflow~=2.6.0",]
+        depconstraint: [
+          "tfx~=1.0.0 tensorflow~=2.5.0",
```

Comment:
nit. Add note to keep in sync w versions in version.py

Reply:
added